Tella/Components/Reports/Draft/DraftView.swift

```diff
@@ -16,7 +16,8 @@ struct DraftView<VM: DraftViewModelProtocol>: View  {
     
     @EnvironmentObject var mainAppModel: MainAppModel
```

Comment:
I think we can remove this  `@EnvironmentObject var mainAppModel: MainAppModel` and use : 
`reportsViewModel.mainAppModel` when it is needed